precise_driver/package.xml

```diff
@@ -12,13 +12,18 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <depend>angles</depend>
+  <depend>actionlib_msgs</depend>
   <depend>cob_srvs</depend>
+  <depend>controller_manager_msgs</depend>
   <depend>geometry_msgs</depend>
+  <depend>message_generation</depend>
```

Comment:
if messages are generated within a package, we have
```
<build_depend>message_generation</build_depend>
<exec_depend>message_runtime</exec_depend>
```
as a special block between `<buildtool_depend>catkin</buildtool_depend>` and the other dependencies